### DISCUSSION TITLE
migrate sqlite to mysql?

### DISCUSSION THREAD

**Discussion by srgvg** [photoprism/photoprism#1214] [Q&A] [answered]
Is it somehow possible to migrate from sqlite to mysql? Has it been done before perhaps?

**Comment by lastzero** [comment, marked as answer]
Yes, but it's out of scope for a photo management solution like ours. We have backup and restore commands. There are external tools available to convert it, although there might be issues as probably not every detail - like indexes - can be migrated. So it might be best to start with an empty index created by PhotoPrism and just migrate the table contents. We can offer paid support if you need this for a commercial project. Otherwise our community can be helpful, some users did this from what I know.
